script/predicate_functions.sh

```diff
@@ -73,22 +69,14 @@ function rspec_rails_compatible {
   fi
 }
 
-function rspec_support_compatible {
-  if [ "$MAINTENANCE_BRANCH" != "2-99-maintenance" ] && [ "$MAINTENANCE_BRANCH" != "2-14-maintenance" ]; then
-    return 0
-  else
-    return 1
-  fi
-}
-
 function additional_specs_available {
   type run_additional_specs > /dev/null 2>&1
   return $?
 }
 
 function documentation_enforced {
   if [ -x ./bin/yard ]; then
-    if is_mri_2plus; then
+    if is_mri; then
```

Comment:
This file is managed by rspec-dev, I'd revert these unless you have a good reason?

Reply:
Sure, makes total sense 👍 
I haven't thought those will be overwritten with the next update from rspec-dev anyway.